io_scene_niftools/modules/nif_import/geometry/vertex/__init__.py

```diff
@@ -45,6 +45,8 @@ class Vertex:
 
     @staticmethod
     def map_vertex_colors(b_mesh, vertex_colors):
+        # [TODO] in Blender 3.2, vertex_colors was deprecated (https://wiki.blender.org/wiki/Reference/Release_Notes/3.2/Python_API)
```

Comment:
should we just upgrade the code and bump the requirement?

Reply:
Yeah, but I think they said somehwere that the new mesh attribute layers api is more performant than the old access

Reply:
After testing, that is indeed faster on the one large mesh I tried it on. That might have to do with the fact that there is less unpacking needed (since it's per-vertex rather than per-face-corner). Anyways, since it's relatively simple, I just added a version check and the new color attribute storage.